ament_flake8/ament_flake8/main.py

```diff
@@ -139,8 +139,10 @@ def generate_flake8_report(config_file, paths, excludes, max_line_length=None):
         return generate_flake8_report(config_file, paths, excludes, max_line_length)
 
     flake8_argv = []
-    flake8_argv.append('--config={0}'.format(config_file))
-    flake8_argv.append('--exclude={0}'.format(excludes))
+    if config_file is not None:
+        flake8_argv.append('--config={0}'.format(config_file))
+    if excludes is not None:
```

Comment:
oh, here's an example of how it can be an empty list:

```
In [1]: import argparse

In [2]: parser = argparse.ArgumentParser()

In [3]: parser.add_argument('--foo', nargs='*')
Out[3]: _StoreAction(option_strings=['--foo'], dest='foo', nargs='*', const=None, default=None, type=None, choices=None, help=None, metavar=None)

In [4]: parser.add_argument('--bar', nargs='*')
Out[4]: _StoreAction(option_strings=['--bar'], dest='bar', nargs='*', const=None, default=None, type=None, choices=None, help=None, metavar=None)

In [5]: parser.parse_args('--foo --bar 1 2'.split())
Out[5]: Namespace(bar=['1', '2'], foo=[])
```

it's strange that argparse has different behaviour base on its position in the list (strange to me, at least!)